{"metadata":{"modules":[{"name":"OpenAI"}],"title":"LocalizedError Implementations","role":"collectionGroup","roleHeading":"API Collection"},"topicSections":[{"generated":true,"title":"Instance Properties","identifiers":["doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/errorDescription","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/failureReason","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/helpAnchor","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/recoverySuggestion"]}],"schemaVersion":{"patch":0,"major":0,"minor":3},"variants":[{"paths":["\/documentation\/openai\/apierrorresponse\/localizederror-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/LocalizedError-Implementations","interfaceLanguage":"swift"},"sections":[],"hierarchy":{"paths":[["doc:\/\/OpenAI\/documentation\/OpenAI","doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse"]]},"kind":"article","references":{"doc://OpenAI/documentation/OpenAI/APIErrorResponse/failureReason":{"type":"topic","role":"symbol","abstract":[],"kind":"symbol","url":"\/documentation\/openai\/apierrorresponse\/failurereason","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/failureReason","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"kind":"identifier","text":"failureReason"},{"text":": ","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"title":"failureReason"},"doc://OpenAI/documentation/OpenAI/APIErrorResponse":{"url":"\/documentation\/openai\/apierrorresponse","kind":"symbol","abstract":[],"title":"APIErrorResponse","fragments":[{"kind":"keyword","text":"struct"},{"text":" ","kind":"text"},{"text":"APIErrorResponse","kind":"identifier"}],"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse","navigatorTitle":[{"text":"APIErrorResponse","kind":"identifier"}],"type":"topic","role":"symbol"},"doc://OpenAI/documentation/OpenAI/APIErrorResponse/errorDescription":{"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/errorDescription","kind":"symbol","role":"symbol","title":"errorDescription","url":"\/documentation\/openai\/apierrorresponse\/errordescription","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"kind":"identifier","text":"errorDescription"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}],"type":"topic","abstract":[]},"doc://OpenAI/documentation/OpenAI":{"abstract":[],"identifier":"doc:\/\/OpenAI\/documentation\/OpenAI","url":"\/documentation\/openai","kind":"symbol","type":"topic","title":"OpenAI","role":"collection"},"doc://OpenAI/documentation/OpenAI/APIErrorResponse/helpAnchor":{"abstract":[],"kind":"symbol","url":"\/documentation\/openai\/apierrorresponse\/helpanchor","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"helpAnchor"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"text":"?","kind":"text"}],"title":"helpAnchor","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/helpAnchor","role":"symbol","type":"topic"},"doc://OpenAI/documentation/OpenAI/APIErrorResponse/recoverySuggestion":{"abstract":[],"title":"recoverySuggestion","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"recoverySuggestion","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"type":"topic","kind":"symbol","url":"\/documentation\/openai\/apierrorresponse\/recoverysuggestion","identifier":"doc:\/\/OpenAI\/documentation\/OpenAI\/APIErrorResponse\/recoverySuggestion"}}}